concordium-consensus/src/Concordium/GlobalState/Persistent/BlockState/Updates.hs

```diff
@@ -1106,6 +1141,116 @@ processTimeParametersUpdates t bu = do
                               pendingUpdates = pendingUpdates{pTimeParametersQueue = SomeParam newpQ}
                             }
 
+-- |Process timeout parameters updates.
+processTimeoutParametersUpdates ::
```

Comment:
Please add a few more details about what it does. Or refer to another similar function which has better documentation.

Reply:
Same goes for the ones below.

Reply:
Added documentation for the ones requested, but I won't be starting documenting the whole module now.